packages/phrases/src/locales/ja/translation/admin-console/api-resources.ts

```diff
@@ -0,0 +1,15 @@
+const api_resources = {
+  page_title: 'APIリソース',
+  title: 'APIリソース',
+  subtitle: '承認されたアプリケーションが利用できるAPIを定義',
+  create: 'APIリソースを作成',
+  api_name: 'API名',
+  api_name_placeholder: 'API名を入力してください',
+  api_identifier: 'API識別子',
+  api_identifier_tip:
+    'APIリソースの一意の識別子です。絶対URIで、フラグメント(#)コンポーネントはありません。OAuth 2.0での<a>resource parameter</a>に等しいです。',
+  api_resource_created: 'APIリソース{{name}}が正常に作成されました',
```

Comment:
I'm wondering if it is possible to let the AI add spaces around western characters when mixing with CJK characters?

Reply:
we should find a proper prompt for this, i've tried several but failed